vectors/nexo/nexo.svg

```diff
@@ -0,0 +1,15 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="48" height="42" fill="none" viewBox="0 0 48 42">
```

Comment:
Same here; Fixed height/width is not allowed as it cannot be converted to PNG easily. I think it can be removed.